packages/backend/src/common/tool/index.ts

```diff
@@ -12,3 +12,20 @@ export const randomEmail = (length = 10, domainLength = 3): string => {
 
 export const getObjectKey = (obj: { [key: string]: any }, path: string): any =>
   path.split('.').reduce((o, i) => o?.[i], obj);
+
```

Comment:
Add a jsdoc comment with a short explanation of what it does + input/return values.